stl/debugger/STL.natvis

```diff
@@ -2322,4 +2322,126 @@ SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
     </Expand>
   </Type>
 
+  <!-- KeyContainer: vector, MappedContainer: vector -->
+  <Type Name="std::flat_map&lt;*,*,*,std::vector&lt;*&gt;,std::vector&lt;*&gt;&gt;">

```

Comment:
I note that these visualizers are not going to work with `vector<bool>`. I don't think we should block the PR because of that, but I can imagine `vector<bool>` being used with `flat_map` so we should probably file an issue to correct the lack of support.

Reply:
That issue is #5037.